teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -388,6 +388,7 @@ interface Build {
     val status: BuildStatus?
     val branch: Branch
     val state: BuildState
+    val personal: Boolean?
```

Comment:
What does `null` mean here? Is such build personal or not?

Reply:
@ARostovsky ?

Reply:
I'm sorry for late response, missed this.
In REST response it can be null and it means build isn't personal. 
![image](https://user-images.githubusercontent.com/16816852/86895953-e9d4e280-c10d-11ea-9cca-a619826f4fe4.png)
And it's always true in REST response if build is personal for sure.
![image](https://user-images.githubusercontent.com/16816852/86895915-d7f33f80-c10d-11ea-9754-ae0cbe012aa3.png)

Reply:
So can we change type of this property to `Boolean` then?

Reply:
True, fixed that